http/mixed-content.json

```
@@ -0,0 +1,322 @@
+{
+  "http": {
+    "mixed_content": {
+      "__compat": {
+        "description": "Blocks some or all insecure mixed content",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Security/Mixed_content",
+        "spec_url": "https://www.w3.org/TR/mixed-content/#intro",
+        "support": {
+          "chrome": {
+            "version_added": "79"
+          },
+          "chrome_android": "mirror",
+          "edge": "mirror",
+          "firefox": {
+            "version_added": "23"
+          },
+          "firefox_android": "mirror",
+          "ie": {
+            "version_added": false
+          },
+          "oculus": "mirror",
+          "opera": "mirror",
+          "opera_android": "mirror",
+          "safari": {
+            "version_added": true
```

This is some version around 9.1 - help determining the right one? There are other cases.

---

I guess we could use ≤ then.

```suggestion
            "version_added": "≤9.1"
```